pkg/ingress/manifest.go

```diff
@@ -26,7 +26,7 @@ import (
 	apisixv1 "github.com/apache/apisix-ingress-controller/pkg/types/apisix/v1"
 )
 
-func diffSSLes(olds,news []*apisixv1.Ssl)(added,updated,deleted []*apisixv1.Ssl){
+func diffSSLes(olds, news []*apisixv1.Ssl) (added, updated, deleted []*apisixv1.Ssl) {
```

Comment:
It's not so symmetrical that you use the plural noun `SSLes` here but has the single noun `SSL` in the data structure`TranslateContext`.

IMHO `SSLes` is so weird and I think just `SSL` is enough.